### ISSUE TITLE
How to scrap metrics from ephemeral and batch jobs with grafana-agent?

### ISSUE THREAD

**Issue by arh** [grafana/agent#2864]
grafana-agent does not support Pushgateway to collect metrics from ephemeral and batch jobs. Therefore, I am using textfile collector from the nodeexporter integration as recommended in [this issue](https://github.com/grafana/agent/issues/557). 

The batch jobs simply create a textfile and nodeexporter scrap that from the file system. However, the metrics are not aggregated when I query them in the grafana dashboard. 

For example, Job1 create a textfile with a Count metric and value 3. Node exporter scrap that and send value of 3 to the grafana dashboard. Then Job2 creates another text file with a Count value of 1 for the same metric. After node exporter scrap that, the grafana dashboard still shows 3 for the value of that metric instead of 4.

```
$cat /etc/grafana-agent.yaml
integrations:
  node_exporter:
    enabled: true
    relabel_configs:
    - replacement: hostname
      target_label: instance
    textfile_directory: /opt/prom_metrics
```

```
$ ls /opt/prom_metrics/
poke_230117_062020.prom  poke_230123_103528.prom  poke_230123_111133.prom  poke_230124_102834.prom  poke_230125_085510.prom  poke_230125_095939.prom
poke_230117_103231.prom  poke_230123_103549.prom  poke_230123_111542.prom  poke_230124_103016.prom  poke_230125_085531.prom  poke_230125_095951.prom
poke_230117_103457.prom  poke_230123_103604.prom  poke_230123_111557.prom  poke_230124_103111.prom  poke_230125_085543.prom  poke_230125_100235.prom
```

**Comment by arh** [comment]
I was under impression that `grafana-agent` is a replacement for Prometheus Server. Therefore, I did not install it and don't have a separate server running that I can push metrics to it. 

Please correct me if my understanding it wrong.

**Comment by rfratto** [comment]
No, you don't need the Prometheus server. Push gateway is completely separate service from the Prometheus server, though.
